pkg/terminal/command.go

```diff
@@ -139,7 +143,7 @@ Print out info for every goroutine. The flag controls what information is shown
 	-g	displays location of go instruction that created the goroutine
 
 If no flag is specified the default is -u.`},
-		{aliases: []string{"goroutine"}, allowedPrefixes: onPrefix | scopePrefix, cmdFn: c.goroutine, helpMsg: `Shows or changes current goroutine
```

Comment:
I don't think the scope prefix functionality should be removed. I think being able to quickly run a single command on a different goroutine/frame can be useful. Also while we never made backwards compatibility claims around the UI I think this is to big of a change.